app/models/group.rb

```diff
@@ -2,6 +2,7 @@ class Group < ApplicationRecord
   belongs_to :gm, class_name: "User", foreign_key: :gm_id
   has_many :player_groups
   has_many :users, through: :player_groups
+  geocoded_by :city
```

Comment:
you might want to wait until we merge https://github.com/Naokimi/rpg-seznamka/pull/12 or implement the changes here, because otherwise this will break

Reply:
![image](https://user-images.githubusercontent.com/3223329/112792746-fdaae500-909e-11eb-9aba-fd978e1fd349.png)